de.manumaticx.pagingcontrol/controllers/widget.js

```diff
@@ -188,6 +194,9 @@ function adjustePositions() {
  * @param {Ti.UI.Scrollableview} scrollable view
  */
 exports.setScrollableView = function(_sv){
+    if($.scrollableView) {
+        throw "Already initialized";
+    }
```

Comment:
the user may want to re-initialize the widget with another instance. why do we need this?